lib/pbench/test/unit/server/query_apis/test_datasets_update.py

```diff
@@ -204,6 +204,8 @@ def fake_bulk(
             (None, "private"),
             (None, "public"),
             ("drb", None),
+            ("test", None),
+            (None, None),
```

Comment:
If you make a change for some other reason, it would be nice if this list of parameterizations were in some sort of order (e.g., all the `"drb"`'s first, then all the `"test"`'s, then the `None`'s; with `"public"`, then `"private"`, then `None` in order for each of them) -- it's a little hard to make sure the list is complete with it jumbled, unless you first figure out the pattern.